packages/mui-material/src/Accordion/Accordion.js

```diff
@@ -133,6 +133,11 @@ const AccordionHeading = styled('h3', {
   all: 'unset',
 });
 
+const AccordionRegion = styled('div', {
+  name: 'MuiAccordion',
+  slot: 'Region',
```

Comment:
```suggestion
  slot: 'Region',
  overridesResolver: (props, styles) => styles.region,
```

Please add test to the describeConfirmance too

Reply:
I thought of adding `overridesResolver` but few of the slots in other components doesn't have this function and even in `Accordion` `heading` slot doesn't have it. So i wasn't sure when to add and when not to add

https://github.com/mui/material-ui/blob/634a83623165598c277adf6c89e1625b2c8f0822/packages/mui-material/src/Accordion/Accordion.js#L129-L135

Reply:
For heading, it was removed in #45970 here https://github.com/mui/material-ui/pull/45970/files#diff-bbc688afec7bc090b1ad7bfd26cc2bedf991eca6fa3d2941b3dcbe04b3ee70a6L132 because the defaultOverridesSlot implements it: https://github.com/mui/material-ui/blob/634a83623165598c277adf6c89e1625b2c8f0822/packages/mui-system/src/createStyled/createStyled.js#L36-L41
So I think even for `region` it is not needed. Also for `region` it is already targeted in AccordionRoot: https://github.com/mui/material-ui/blob/master/packages/mui-material/src/Accordion/Accordion.js#L43